cdci_data_analysis/analysis/parameters.py

```diff
@@ -563,9 +578,14 @@ def __init__(self, value=None, units=None, name=None):
                          name=name,
                          allowed_units=None)
 
-    def get_value_in_default_format(self) -> Union[str, float, None]:
+    def get_value_in_default_units(self) -> Union[str, float, None]:
         return getattr(self._astropy_angle, self.default_units)
 
+    @staticmethod
```

Comment:
Why not instance method?

Reply:
not a particular reason, but we have used `staticmethod` also within the rest of the class

Reply:
also, the intention here is to have a utility function, so `staticmethod` makes sense here

Reply:
I am asking just because it is way prettier to use something like `parameter.value_in_units('arcsec')` if one wants to pass the parameter to backend in these units, instead of using internal representation like `parameter._astropy_angle.arcsec` or `Angle.get_value_in_units(parameter.value, parameter.default_units, parameter.units)`